protocols/ping/src/handler.rs

```diff
@@ -18,10 +18,11 @@
 // FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
 // DEALINGS IN THE SOFTWARE.
 
-use crate::protocol;
+use crate::{protocol, PROTOCOL_NAME};
 use futures::future::BoxFuture;
 use futures::prelude::*;
 use futures_timer::Delay;
+use libp2p_core::upgrade::ReadyUpgrade;
```

Comment:
Given that `libp2p-ping` now depends on `libp2p-core` `v0.35.2`, the `libp2p-ping` `Cargo.toml` needs to be adjusted, right?

Reply:
Good catch!